spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -339,7 +339,7 @@ public String toString() {
 
 		private int count;
 
-		private long last;
+		private long last = System.currentTimeMillis();
```

Comment:
Any explanation for this change, please?

Reply:
Sorry - yes; I just noticed some NPEs in tests due to the scheduled `checkConsumer()` running too early.